.gitignore

```diff
@@ -0,0 +1 @@
+.sass-cache
```

Comment:
It's common to set up a `.gitignore` file to ignore files that we don't want in our repos.  .sass-cache (or any cache) is a good idea.  Also, it's a common practice to add your `bower_components` and `node_modules` to the `.gitignore` so you aren't checking your dependencies.  Then, when someone installs your app, all they have to do is run `npm install && bower install` and they'll get all the dependencies.